bbc/grid
# 1194 interim filters

## Purpose of PR/change

This change introduces the interim filters design for BBC Images version of Grid. It creates a two tier top bar and move the filter controls into the lower bar. The filter drop down choices add/remove chips from the search bar and execute the chosen search. The chargeable toggle controls visibility of chargeable images.

This changes is hidden behind a feature flag set in the kahuna config and when turned off the UI should remain unchanged.

![Show payable images toggle on](https://github.com/bbc/grid/assets/128470622/97e7dc49-6ac7-453b-94e0-61add55ab2d9)

## How should a reviewer test this change?
Ensure that the drop down options produce the correct chip settings and that the filter is executed correctly.
Ensure that when the feature flag is off the original UI operates as expected.

## Tested? Documented?
